### ISSUE TITLE
[Accessibility] Accessibility focus moves inappropriatley to the code editor section.

### ISSUE THREAD

**Issue by Krituhar** [microsoft/vscode#227265]
- VS Code Version: 1.88.0
- OS Version: MacOS

**Issue**: When loading the VSCode iframe in Lambda Console, the accessibility focus moves inappropriately to the code editor section, instead of top of the page.

**Impact**: Screen reader users and screen magnification users may become disoriented. Keyboard users will have to navigate through the contents of the page.

**Steps to Reproduce:**
1. Have the accessibility screen reader open.
2. On a webpage load with VSCode **iframe** loading from a CDN
3. The screen reader focus automatically moves inappropriately to the code editor section instead of top of page.
4. VSCode is somewhere intercepting the accessibility focus to go straight to editor section instead of top of page. We don't have this issue currently with Cloud9 in lambda console.

**[Recommendation]**
Ensure content updates define focus updates appropriately. When a link is activated, focus should be on the top of the page. Use the JavaScript focus() method to move keyboard focus to this element.

**Comment by Jasomadrid** [comment]
> * VS Code Version: 1.88.0
> * OS Version: MacOS
> 
> **Issue**: When loading the VSCode iframe in Lambda Console, the accessibility focus moves inappropriately to the code editor section, instead of top of the page.
> 
> **Impact**: Screen reader users and screen magnification users may become disoriented. Keyboard users will have to navigate through the contents of the page.
> 
> **Steps to Reproduce:**
> 
> 1. Have the accessibility screen reader open.
> 2. On a webpage load with VSCode **iframe** loading from a CDN
> 3. The screen reader focus automatically moves inappropriately to the code editor section instead of top of page.
> 4. VSCode is somewhere intercepting the accessibility focus to go straight to editor section instead of top of page. We don't have this issue currently with Cloud9 in lambda console.
> 
> **[Recommendation]** Ensure content updates define focus updates appropriately. When a link is activated, focus should be on the top of the page. Use the JavaScript focus() method to move keyboard focus to this element.
> 
> PFA recording of the issue. [Screen Recording 2024-08-30 at 5.42.39 PM.zip](https://github.com/user-attachments/files/16822958/Screen.Recording.2024-08-30.at.5.42.39.PM.zip)

**Comment by Krituhar** [comment]
Creating a new one
